portfolio/src/main/webapp/style.css

```diff
@@ -96,6 +96,11 @@ a {
   grid-gap: 10px;
 }
 
-.random-quote-container{
+.random-quote-container {
   font-size: 25px;
 }
+
+#HoustonMap {
```

Comment:
use lowercase for html/css `#HoustonMap` --> `#houston-map`

https://google.github.io/styleguide/htmlcssguide.html#Capitalization